ballsdex/packages/players/cog.py

```diff
@@ -111,6 +135,243 @@ async def delete(self, interaction: discord.Interaction):
         await player.delete()
 
+    @friend.command()
+    async def add(self, interaction: discord.Interaction, user: discord.User):
+        """
+        Add another user as a friend.
+
+        Parameters
+        ----------
+        user: discord.User
+            The user you want to add as a friend.
+        """
+        player1, _ = await PlayerModel.get_or_create(discord_id=interaction.user.id)
+        player2, _ = await PlayerModel.get_or_create(discord_id=user.id)
+
+        if player1 == player2:
+            await interaction.response.send_message(
+                "You cannot add yourself as a friend.", ephemeral=True
+            )
+            return
+        if user.bot:
+            await interaction.response.send_message("You cannot add a bot.", ephemeral=True)
+            return
+
+        blocked = await player1.is_blocked(player2)
+
+        if blocked:
+            await interaction.response.send_message(
+                "You cannot add a blocked user. To unblock, use `/player unblock`.",
+                ephemeral=True,
+            )
+            return
+
+        friended = await player1.is_friend(player2)
+        if friended:
+            await interaction.response.send_message(
+                "You are already friends with this user!", ephemeral=True
+            )
+        else:
+            await Friendship.create(player1=player1, player2=player2)
+            await interaction.response.send_message(
+                f"You are now friends with {user.name}.", ephemeral=True
+            )
+
+    @friend.command()
+    async def remove(self, interaction: discord.Interaction, user: discord.User):
+        """
+        Remove a friend.
+
+        Parameters
+        ----------
+        user: discord.User
+            The user you want to remove as a friend.
+        """
+        player1, _ = await PlayerModel.get_or_create(discord_id=interaction.user.id)
+        player2, _ = await PlayerModel.get_or_create(discord_id=user.id)
+
+        if player1 == player2:
+            await interaction.response.send_message("You cannot remove yourself.", ephemeral=True)
+            return
+        if user.bot:
+            await interaction.response.send_message("You cannot remove a bot.", ephemeral=True)
+            return
+
+        friendship_exists = await player1.is_friend(player2)
+        if not friendship_exists:
+            await interaction.response.send_message(
+                "You are not friends with this user.", ephemeral=True
+            )
+            return
+        else:
+            await Friendship.filter(
+                (Q(player1=player1) & Q(player2=player2)) |
+                (Q(player1=player2) & Q(player2=player1))
+            ).delete()
+            await interaction.response.send_message(
+                f"{user.name} has been removed as a friend.", ephemeral=True
+            )
+
+    @friend.command()
+    async def list(self, interaction: discord.Interaction):
```

Comment:
```suggestion
    @friend.command(name="list")
    async def friend_list(self, interaction: discord.Interaction):
```

Using the same function names can result in some weird errors/handling, this should be done for all where the func names overlap



ie: add, remove and list for friend and block